spec/registry.md

```diff
@@ -102,11 +106,11 @@ For the sake of brevity, only `locales="en"` is considered.
             <option name="minimumSignificantDigits" validationRule="positiveInteger"/>
             <option name="maximumSignificantDigits" validationRule="positiveInteger"/>
             <!-- Since this applies to both cardinal and ordinal, all plural options are valid. -->
-            <match values="zero one two few many"/>
-            <match validationRule="anyNumber"/>
+            <match locales="en" values="one two few other" validationRule="anyNumber"/>
+            <match values="zero one two few many other" validationRule="anyNumber"/>
```

Comment:
`other` or `*`?

---

`locales="en"` looks odd. Can `locales` contain multiple values? Space separated, I guess? 

This has always struck me as a wobbly mechanism. Many locales have identical plural rules to English. Are we going to list them? Why do we need to repeat CLDR data here? Perhaps we should have a referencing mechanism to CLDR instead of replicating data. Note that other things besides plurals depend on CLDR data, such as date patterns and such.